projects/1.18/assets/chunk-loaders/chunkloaders/lang/zh_cn.json

```
@@ -1,10 +1,28 @@
 {
-  "block.chunkloaders.single_chunk_loader": "单区块加载器",
-  "block.chunkloaders.basic_chunk_loader": "基础区块加载器",
-  "block.chunkloaders.advanced_chunk_loader": "高级区块加载器",
-  "block.chunkloaders.ultimate_chunk_loader": "终极区块加载器",
+  "chunkloaders.gui.loaded_chunks": "已加载区块：",
+  "chunkloaders.gui.loaded_chunks.count": "%s",
+  "chunkloaders.keys.category": "区块加载器",
   "chunkloaders.gui.loaded": "已加载",
-  "chunkloaders.gui.unloaded": "未加载",
+  "chunkloaders.block.single_chunk_loader": "单区块加载器",
+  "chunkloaders.gui.disabled": "该服务器已禁用区块加载器映射图",
+  "chunkloaders.legacy_message": "请尝试Shift-右击该区块加载器来设置其持有者！",
+  "chunkloaders.gui.owner": "持有人：",
   "chunkloaders.chunk_loader.info.single": "可以加载单个区块",
-  "chunkloaders.chunk_loader.info.multiple": "最多可加载 %1$d X %1$d 个区块"
-}
\ No newline at end of file
+  "chunkloaders.gui.speech.chunk.others": "由其他玩家加载",
+  "chunkloaders.chunk_loader.info.multiple": "最多可加载 %1$d x %1$d 个区块",
+  "chunkloaders.gui.unloaded": "未加载",
+  "chunkloaders.gui.chunk.overwrite": "按住%s进行覆写",
+  "chunkloaders.legacy_success": "你现在是该区块加载器的持有者了！",
+  "chunkloaders.gui.chunk.others": "加载者为",
+  "chunkloaders.block.ultimate_chunk_loader": "终极区块加载器",
+  "chunkloaders.gui.loaded_chunks.count_max": "%1$s / %2$s",

```

```suggestion
  "chunkloaders.gui.loaded_chunks.count_max": "%1$s/%2$s",
```
具体应以游戏内显示是否清晰为准

在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中**全部查明并修改**。

---

这里是无翻译的内容，是否直接删除该中文字段会更好一些

---

别删，建议中英文行数完全一致